### ISSUE TITLE
tried to convert t to a tensor and failed error none values not supported

### ISSUE THREAD

**Issue by zhangjinyangnwpu** [tensorflow/tensorflow#21026]
When I want to set a trainable variable, and use 
```
alpha = tf.Variable(initial_value=1.0)
beta= tf.Variable(initial_value=1.0)

loss = alpha*loss_1 + beta*loss_2
```
and use it with a gradient clip, problem happened,
errors reported like: 
```
tried to convert t to a tensor and failed error none values not supported
```
What should I do?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by tanzhenyu** [comment]
Do you have a code snippet?

**Comment by zhangjinyangnwpu** [comment]
Thanks, I have solved this problem by using 
```
alpha = tf.Variable(initial_value=tf.Constant(1.0))
beta= tf.Variable(initial_value=tf.Constant(1.0))
loss = alpha*loss_1 + beta*loss_2
```
